server/server.go

```diff
@@ -32,13 +31,12 @@ func New(bindAddr string, router http.Handler) *Server {
 	middleware := map[string]alice.Constructor{
 		"RequestID": requestID.Handler(16),
 		"Log":       log.Handler,
-		"Timeout":   timeout.Handler(10 * time.Second),
 	}
 
 	return &Server{
 		Alice:           nil,
 		Middleware:      middleware,
-		MiddlewareOrder: []string{"RequestID", "Log", "Timeout"},
+		MiddlewareOrder: []string{"RequestID", "Log"},
```

Comment:
Could both "RequestID" and Log" be constants?

Reply:
Done